lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -626,18 +668,18 @@ migrateManually tr proxy defaultFieldValues =
             ColumnPresent ->
                 traceWith tr $ MsgManualMigrationNotNeeded field
 
-    renameColumn
+    renameColumnField
         :: Sqlite.Connection
         -> DBField
         -> Text -- Old Value
         -> Text -- New Value
         -> IO ()
-    renameColumn conn field old new = do
+    renameColumnField conn field old new = do
         isFieldPresent conn field >>= \case
             TableMissing ->
                 traceWith tr $ MsgManualMigrationNotNeeded field
             ColumnMissing -> do
-                traceWith tr $ MsgManualMigrationNotNeeded field
+                traceWith tr $ MsgManualMigrationNeeded field new
```

Comment:
ah, changed this `msg` but should be in `RenameRoleColumn` :thinking:

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/2308/commits/3f6a16584b53c5f34581f7012ac5bbff69c5f126